commcare_connect/opportunity/forms.py

```diff
@@ -860,6 +860,7 @@ def __init__(self, *args, **kwargs):
             Row(Field("amount")),
             Row(Field("date")),
             Row(Field("invoice_number")),
+            Row(Field("other_expense")),
```

Comment:
I think we want to invert all this and default to true and call it "Service Delivery," it is unclear otherwise what "other" refers to (ie what it is other than)

Reply:
Done. [05468b3](https://github.com/dimagi/commcare-connect/pull/491/commits/05468b328d14ff3b9114b83855d3a75b4cfc19b4)